src/HomeKit/HMEnums.cs

```diff
@@ -208,7 +208,18 @@ public enum HMCharacteristicType {
 		TargetPosition,
 		[iOS (9,0)]
 		TargetVerticalTilt,
-
```

Comment:
Add the availability attributes on the members since they will be used as an input to an existing API (and things might go wrong as the conversion will turn them into `null`).